rs-digital-menu/src/pages/Home.js

```diff
@@ -55,6 +66,14 @@ const Home = () => {
           Pastelería especializada en la creación de tartas matrimoniales
         </p>
         <h2 className="mt-4 text-lg text-lato font-medium text-custom-primary">Categorías</h2>
+        <Authorize requireAdmin={false}>
```

Comment:
@Diego-Oz remove the requireAdmin={false} as all the admin functionallity should requiere the admin role.